src/medigan/install_model_dependencies.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import subprocess
+import sys
+
+from medigan.config_manager import ConfigManager
+from medigan.constants import CONFIG_FILE_KEY_DEPENDENCIES, CONFIG_FILE_KEY_EXECUTION
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--model_id",
+        type=str,
+        default=None,
+        nargs="+",
+        help="Model ids to install dependencies for",
+    )
+    args = parser.parse_args()
+    return args
+
+
+def install_model(model_id):
```

Comment:
For security purposes, I think we should prompt the user to confirm they want us to install a model's packages to their system. This will allow us to integrate the function `def install_model(model_id):` into medigan.model_executor

I would suggest do add a parameter `is_user_prompt` to `def install_model(model_id, is_user_prompt:bool = True):` that is only set to false when the function is called from `"__main__"` of `src/medigan/install_model_dependencies.py`

We could create another PR for this, but maybe it is easier to just add it here before it reaches the main branch.
Maybe something along the lines of:
`if "YES" == input( f"Install the dependencies {dependencies} of model {model_id} programatically? Type YES, to proceed: "):`

Reply:
I started adding it, but then some additional issues occurred. During tests or some automated runs for example, it is quite problematic to require user action. I think it is enough to receive a message with clear instructions what command needs to be executed in order to install the dependencies. I can do that in the separate PR if we have some consensus.